### ISSUE TITLE
epub3-to-pef

### ISSUE THREAD

**Issue by bertfrees** [daisy/pipeline-mod-braille#20]
Make a EPUB3 to PEF script.

Duplicate of: https://github.com/snaekobbi/pipeline-mod-braille/issues/43

**Comment by josteinaj** [comment]
blocked on: #30

**Comment by bertfrees** [comment]
Closing because duplicate
